### ISSUE TITLE
@azure/msal-angular on .NET Core Project. Token Timeout prevents navigation [MsalGuard]

### ISSUE THREAD

**Issue by HockeyJustin** [AzureAD/microsoft-authentication-library-for-js#1295]
## Library
- [X ] `@azure/msal@1.2.2-beta.0`

## Framework
Angular 8

## Description
This is similar to [this issue](https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/1139#issuecomment-580788098). That issue caused an error on accessing the API, which has now been fixed. 

This issue: An old token allows access to resources, but renders the angular application unable to navigate `MsalGuard`-ed resources.

## Security
No

## Regression
No

## Configuration
Standard - as with the Angular 8 sample. Accessing a local API protected resource. 

```js
export const protectedResourceMap: [string, string[]][] = [
	['http://localhost:8973/api/Values', ['https://[redacted].onmicrosoft.com/api/demo.read']],
	['https://graph.microsoft.com/v1.0/me', ['user.read']]];

const isIE = window.navigator.userAgent.indexOf("MSIE ") > -1 || window.navigator.userAgent.indexOf("Trident/") > -1;


@NgModule({
	declarations: [
		AppComponent,
		HomeComponent,
		LogoutComponent,
		ValuesComponent
	],
	imports: [
		BrowserModule,
		AppRoutingModule,
		HttpClientModule,
    MsalModule.forRoot({
      auth: {
        clientId: 'XXXXXXXX-XXXX-XXX-XXXX-XXXXXXXXX',
        authority: "https://[redacted].b2clogin.com/[redacted].onmicrosoft.com/B2C_1_signupsignin1",
        validateAuthority: false,
        redirectUri: "http://localhost:8744/app/dist/app/home",
        postLogoutRedirectUri: "http://localhost:8744/app/dist/app/home",
        navigateToLoginRequestUrl: true,
      },
      cache: {
        cacheLocation: "localStorage",
        storeAuthStateInCookie: isIE, // set to true for IE 11
      },
    },
      {
        popUp: false,
        consentScopes: [
          "https://[redacted].onmicrosoft.com/api/demo.read"
        ],
        unprotectedResources: ["https://www.microsoft.com/en-us/"],
        protectedResourceMap,
        extraQueryParameters: {}
      })
	],
	providers: [
		{
			provide: HTTP_INTERCEPTORS,
			useClass: MsalInterceptor,
			multi: true
		},
		{
			provide: HTTP_INTERCEPTORS,
			useClass: HttpErrorInterceptorService,
			multi: true,
		}
	],
	bootstrap: [AppComponent]
})
export class AppModule { }
```

## Reproduction steps
1. Create a MsalAngular app (in line with Angular 8 sample) with a local API to access
2. Login, call the api, then leave long enough for the token to expire
3. Call your api (optional, but I did this)
4. Try to navigate around the application.

## Expected behavior
The application bounces to an unguarded resource and is logged out, or preferably, a new token continues to allow navigation. 

NOTE: After the wait, you can access your API, but you can't navigate around the app.

## Browsers
No

**Comment by jcox86** [comment]
Also have this issue on `Angular 8.2.0` and `@azure/msal-angular 0.1.2`.

**Comment by jasonnutter** [comment]
This should be fixed by #1455.
